apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/DexController.java

```diff
@@ -266,6 +266,10 @@ public Response cancelOrderByOrderID(@Parameter(description = "Order id") @FormP
                 return Response.status(Response.Status.OK).entity(JSON.toString(incorrect("orderId", "Can cancel only Open orders."))).build();
             }
 
+            if(service.isThereAnotherCancelUnconfirmedTr(transactionId, null)){
+                return Response.status(Response.Status.OK).entity(JSON.toString(incorrect("orderId", "There is another cancel transaction for this order in the pool already."))).build();
+            }
+
 
             CustomRequestWrapper requestWrapper = new CustomRequestWrapper(req);
             requestWrapper.addParameter("deadline", Constants.GUARANTEED_BALANCE_CONFIRMATIONS.toString());
```

Comment:
Create separate constant for deadline. GuaranteedBalanceConfirmations is not referred to dex